ERP-BACKEND/app/routers/health.py

```
@@ -0,0 +1,378 @@
+        
+        missing_tables = [t for t in expected_tables if t not in actual_tables]
+        extra_tables = [t for t in actual_tables if t not in expected_tables and not t.startswith("alembic")]
+        
+        # Check indexes
+        index_issues = []
+        for table_name in expected_tables:
+            if table_name in actual_tables:
+                try:
+                    indexes = inspector.get_indexes(table_name)
+                    # Could add specific index validation here
+                except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)